modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in all namespaces. After the objects are removed, you can then remove the Operator and its namespace.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The File Integrity Operator is installed.
+
+.Procedure
+
+. Navigate to the *Operators* -> *Installed Operators* -> File Integrity Operator page.
+
+. Use the default option *Show operands in: All namespaces* to list all `FileIntegrity` objects in all namespaces.
+
+. Click the Options menu {kebab} and use it to delete all `FileIntegrity` objects.
```

Comment:
```suggestion
. Click the *Options* menu {kebab} and then click *Delete FileIntegrity* to delete a `FileIntegrity` object.
```

I'm unsure if "Options" should be capitalized as it's not the official name of the menu. Maybe lowercase it?

![Screenshot From 2025-04-09 11-06-52](https://github.com/user-attachments/assets/d84bf06f-d69e-4003-9eeb-60fae5fa7f52)

Also does a customer have to click the button each time to delete an object?

Reply:
We do not bold Options here and seem to capitalize it nearly all of the time.  The {kebab} add an image to the step. See [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#inserting-reusable-images-inline).